manifests/vhost.pp

```diff
@@ -385,6 +386,13 @@
     }
   }
 
+  # Load mod_passenger if needed and not yet loaded
+  if $passenger_base_uris {
+    if ! defined(Class['apache::mod::passenger']) {
```

Comment:
You don't need to have this check, `include ::apache::mod::passenger` will just work.